charts/tidb-operator/values.yaml

```diff
@@ -112,6 +112,8 @@ controllerManager:
   # runAsUser: 1000
   # runAsGroup: 2000
   # fsGroup: 2000
+  podAnnotations: {}
+  # Refer to https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
```

Comment:
We often put the refer above the fields, ie

```yaml
  # Refer to https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
  podAnnotations: {}
```

Reply:
sure